javascript/javascript3/week3/shopping-cart.js

```diff
@@ -0,0 +1,100 @@
+const userName = document.querySelector('h2');
+const ul = document.querySelector('ul');
+
+class Product {
+    constructor(name, price) {
+        this.name = name;
+        this.price = price;
+    }
+    convertToCurrency(currency) {
+        if (currency === "dollars") {
+            const dkkToDollar = 6.28;
+            this.price = this.price / dkkToDollar;
+            return this.price;
+        } else if (currency === "pound") {
+            const dkkToPound = 8.62;
+            this.price = this.price / dkkToPound;
+            return this.price;
+        } else if (currency === "euro") {
+            const dkkToEuro = 7.43;
+            this.price = this.price / dkkToEuro;
+            return this.price;
+        }
+    }
+}
+
+class ShoppingCart {
+    constructor() {
+        this.products = [];
+    }
+    addProduct(product) {
+        this.products.push(product);
+    }
+    removeProduct(product) {
+        const index = this.products.indexOf(product);
+        this.products.splice(index, 1);
+    }
+    searchProduct(productName) {
+        this.products.forEach((items) => {
+            if (items.name == productName) {
+                console.log(`${items.name} is in your list`);
+            }
+        })
+    }
+    getTotal() {
+        let sum = 0;
+        this.products.forEach(element => {
+            sum += element.price;
+        });
+        return sum;
+    }
+    renderProducts() {
+        this.getUser().then((data) => userName.innerHTML = data.name);
+        this.products.forEach(element => {
+            const li = document.createElement('li');
+            ul.appendChild(li);
+            const h2 = document.createElement('h3');
+            h2.innerHTML = element.name;
```

Comment:
you are creating h3 but your variable name id h2. It should be same for better naming convention.

Reply:
Thanks Swapna for the feedback.